### PR TITLE
I18n phrase ja

### PR DESCRIPTION
**Pull request by iaraltkorn**
### Summary

My PR...

### Related PRs, issues, or features (optional)

- N/A

### Feature release date (optional)

- N/A

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Comment by iaraltkorn** [comment]
@braze-inc/docs-team can you help with this? i resolved the issues and now I'm getting an error in one of the checks

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by josh-mccrowell-braze** [comment]
So, @iaraltkorn , we're not entirely sure what's happening here but as a near term fix, please fill out the information [here](https://www.braze.com/docs/cla/) using this username. Once done, you can type "@cla-bot check" as a comment to see if it worked. Finger's crossed!

File: _lang/ja/_partners/message_personalization/dynamic_content/movable_ink.md [hunk 1]
```diff
@@ -2,176 +2,177 @@
...
+
+### AndroidでGIFを表示するのに問題がありますか?
+- Androidには、実施にあたってGIFの支援が必要である。この設定がない場合は、Android\[アプリ内メッセージ customization]\[GIFsupport] 記事]に従ってください。
 
 [1]: https://www.movableink.com/
-[データソース]: ({% image_buster /assets/img/movable_ink/movable_ink1.png %})
-[サポート]: https://support.movableink.com/
-[GIFサポート]: {{site.baseurl}}/developer_guide/platform_integration_guides/android/in-app_messaging/customization/gifs/
-[指示]: {{site.baseurl}}/partners/message_personalization/dynamic_content/movable_ink/#step-4-braze-experience
+\[データソース]: ({% image_buster /assets/img/movable_ink/movable_ink1.png %})
```
Comment (josh-mccrowell-braze) [inline _lang/ja/_partners/message_personalization/dynamic_content/movable_ink.md:172]:
```suggestion
[データソース]: ({% image_buster /assets/img/movable_ink/movable_ink1.png %})
```

File: _lang/ja/_partners/message_personalization/dynamic_content/movable_ink.md [hunk 2]
```diff
@@ -2,176 +2,177 @@
...
+サポート]: https://support.movableink.com/
```
Comment (josh-mccrowell-braze) [inline _lang/ja/_partners/message_personalization/dynamic_content/movable_ink.md:173]:
```suggestion
[サポート]: https://support.movableink.com/
```

File: _lang/ja/_partners/message_personalization/dynamic_content/movable_ink.md [hunk 3]
```diff
@@ -2,176 +2,177 @@
...
+\[GIFsupport]: {{site.baseurl}}/developer_guide/platform_integration_guides/android/in-app_messaging/customization/gifs/
```
Comment (josh-mccrowell-braze) [inline _lang/ja/_partners/message_personalization/dynamic_content/movable_ink.md:174]:
```suggestion
[GIFsupport]: {{site.baseurl}}/developer_guide/platform_integration_guides/android/in-app_messaging/customization/gifs/
```

File: _lang/ja/_partners/message_personalization/dynamic_content/movable_ink.md [hunk 4]
```diff
@@ -2,176 +2,177 @@
...
+\[指示]: {{site.baseurl}}/partners/message_personalization/dynamic_content/movable_ink/#step-4-braze-experience
```
Comment (josh-mccrowell-braze) [inline _lang/ja/_partners/message_personalization/dynamic_content/movable_ink.md:175]:
```suggestion
[指示]: {{site.baseurl}}/partners/message_personalization/dynamic_content/movable_ink/#step-4-braze-experience
```

File: _lang/ja/_user_guide/data_and_analytics/export_braze_data/export_campaign_results_data.md [hunk 5]
```diff
@@ -73,19 +73,19 @@ tool:
...
 権限がないと、\[**ユーザーデータ**] ボタンは表示されません。ユーザーデータをエクスポートするには、そのワークスペースの**ユーザーデータのエクスポート**[権限]({{site.baseurl}}/user_guide/administrative/manage_your_braze_users/user_permissions/#limited-and-team-role-permissions)が必要です。
 {% endalert %}
 
-![キャンペーン詳細ページのユーザーデータドロップダウン][6]
+![キャンペーン詳細ページの「ユーザデータ」ドロップダウン][6]
 
 CSV 出力には、キャンペーンの各受信者のユーザープロファイルデータが含まれます。Braze はバックグラウンドでレポートを生成し、現在ログインしているユーザーにメールで通知します。
 
-Amazon S3 credentials][26] ] をBrazeにリンクしている場合、CSVはS3バケットにもアップロードされる。それ以外の場合、メールのリンクは数時間で期限切れになります。
+\[Amazon S3 認証情報 s][26] を Braze にリンクしている場合、CSV も S3 バケットにアップロードされます。それ以外の場合、メールのリンクは数時間で期限切れになります。
```
Comment (josh-mccrowell-braze) [inline _lang/ja/_user_guide/data_and_analytics/export_braze_data/export_campaign_results_data.md:86]:
```suggestion
[Amazon S3 認証情報 s][26] を Braze にリンクしている場合、CSV も S3 バケットにアップロードされます。それ以外の場合、メールのリンクは数時間で期限切れになります。
```

**Review by josh-mccrowell-braze — APPROVED** [review]

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Pull request merged**